lms/urls.py

```diff
@@ -78,6 +78,12 @@
 
 )
 
+if settings.SITE_NAME == 'suclass.stanford.edu':
```

Comment:
hey @kluo .  sorry, after a bit of discussion, @stvstnfrd and I think we should just create another setting, called SHIB_ONLY_SITE (and set it to false in envs.py and add an override in aws.py).  And condition this based on that setting.